### DISCUSSION TITLE
RTK Optimistic Updates in "Select" element default value not working without refreshing page

### DISCUSSION THREAD

**Discussion by rosgeorg** [reduxjs/redux-toolkit#4325] [Q&A]
Hello, I'm facing this situation where I'm trying to modify the default value of a select (dropdown) element using optimistic updates, but even when my query's cache is correctly updated after mutation, the default value of this select won't change until I refresh the page. Is there a way to fix this?

I also added a "div" with the same data inside of it, and it get's updated correctly inside of the div, but not in the default value
 
<img width="803" alt="Screenshot 2024-04-04 at 7 44 15 p m" src="https://github.com/reduxjs/redux-toolkit/assets/15363871/a4f24607-c637-48ff-9b2c-e84861e799aa">

Thanks in advance for your help

**Comment by markerikson** [comment]
`defaultValue` is just that - the _initial_ value. It doesn't control the value in the future.  Do you need to change that to `value=` instead?
